docs/esp8266/quickref.rst

```diff
@@ -409,6 +409,17 @@ The DHT driver is implemented in software and works on all pins::
     d.temperature() # eg. 23.6 (°C)
     d.humidity()    # eg. 41.3 (% RH)
 
+WDT (Watchdog timer)
```

Comment:
would be good grouped with other machine module items

Reply:
@dpgeorge Done. Moved higher up.